__tests__/app.test.js

```diff
@@ -92,3 +90,34 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("GET:200 Responds with an articles array of article objects, each of which should have have appropriate properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeSortedBy("created_at", { descending: true });
+        body.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET:404 Responds with a 404 error message for  incorrect path", () => {
+    return request(app)
+      .get("/api/artcles")
```

Comment:
this is a good test but more generic for the whole api.  so maybe put in a different describe block.